rover:
  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 10
  joint_wheel_front_right_velocity_controller:
    type: velocity_controllers/JointVelocityController
    joint: joint_wheel_front_right
  joint_wheel_front_left_velocity_controller:
    type: velocity_controllers/JointVelocityController
    joint: joint_wheel_front_left
  joint_wheel_back_right_velocity_controller:
    type: velocity_controllers/JointVelocityController
    joint: joint_wheel_back_right
  joint_wheel_back_left_velocity_controller:
    type: velocity_controllers/JointVelocityController
    joint: joint_wheel_back_left
